.github/workflows/build.yml

```diff
@@ -2,66 +2,61 @@ name: Build
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
-
-  build:
-    name: Build and Test
+  build_and_test:
+    name: Build and test
```

Comment:
Aren't we naming it as "Build and Test" in other repos?

Reply:
https://github.com/konveyor/move2kube/blob/17f03ff1498fb16503fd7b8411410d0b6afc1e90/.github/workflows/prbuild.yml#L10

Reply:
Yeah I was working backwards from `move2kube-tests` move2kube-operator` `move2kube-ui` `move2kube-api`
Will get to `move2kube`